server/csharp/TicketHub/Authorization/OpaAuthzService.cs

```diff
@@ -6,10 +6,11 @@ namespace TicketHub.Authorization;
 public class OpaAuthzService
 {
     private readonly OpaClient opa;
-    public OpaAuthzService()
+    public OpaAuthzService(ILogger<OpaClient> logger)
     {
         string opaUrl = Environment.GetEnvironmentVariable("OPA_URL") ?? "http://localhost:8181";
-        opa = new OpaClient(opaUrl);
+        opa = new OpaClient(opaUrl, logger);
+        logger.LogInformation("OPA Client initialized");
```

Comment:
This small addition ensures we'll see an Info-level logged message when the OpaClient comes online. I found this useful for checking that the logger was wired in correctly by DI.